proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -59,12 +59,10 @@ message OrganizationUser {
 // These preferences are stored in the database so the language and plugins
 // are pre-selected when the user navigates back to the Generated SDKs page from any device or browser.
 message UserPluginPreference {
-  // The id of the user.
-  string user_id = 1;
   // The language that the user has indicidated as their preferred language for plugins.
-  PluginLanguage selected_language = 2;
+  PluginLanguage selected_language = 1;
```

Comment:
As a general thing, you shouldn't do this - this is backwards-incompatible. There's no downside to using `reserved 1` and `reserved "user_id"`